src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="warn" name="SupportBank" packages="">
+    <Appenders>
+        <File name="FileAppender" fileName="logs/BusBoard.log">

```

Comment:
We were wondering about why Heroku wasn't showing the application logs, and I think this is the reason - we're configuring log4j to save logs to a file that Heroku doesn't know about.

To get the logs into Heroku, I think you'd need to add another `Appender` which logs to the standard output. Heroku was picking up `System.out.println` logs so we know it is reading stdout. See the `ConsoleAppender` config here: https://logging.apache.org/log4j/2.x/manual/appenders.html#ConsoleAppender